src/App.tsx

```diff
@@ -1,29 +1,57 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [currentPage, setcurrentPage] = useState<number>(1);
+  const itemsPerPageOptions = [3, 5, 10, 20];
+  const [itemsPerPage, setItemsPerPage] = useState<number>(5);
```

Comment:
No need to re-create this variable on each render
```suggestion
const itemsPerPageOptions = [3, 5, 10, 20];

export const App: React.FC = () => {
  const [currentPage, setcurrentPage] = useState<number>(1);
  const [itemsPerPage, setItemsPerPage] = useState<number>(5);
```